shared/common-adapters/radio-button.native.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import React from 'react'
+import ClickableBox from './clickable-box'
+import Text from './text'
+import type {Props} from './radio-button'
+import {globalStyles, globalColors} from '../styles'
+
+export const RADIOBUTTON_SIZE = 22
+export const RADIOBUTTON_MARGIN = 8
+
+const RadioButton = ({disabled, label, onSelect, selected, style}: Props) => {
+  const boxStyle = {
```

Comment:
should use glamorous or at least pull the static things into a const at the bottom and just define the dynamic stuff here like

```
const boxStyle = {
  ..._baseBaseStyle,
  opacity: disabled ? 0.4 : 1,
borderColor: // etc
}
```